ssl/statem/extensions.c

```diff
@@ -554,33 +553,33 @@ static int verify_extension(SSL_CONNECTION *s, unsigned int context,
 int extension_is_relevant(SSL_CONNECTION *s, unsigned int extctx,
                           unsigned int thisctx)
 {
-    int is_tls13;
+    int is_tls13_or_dtls13;
```

Comment:
This is really wordy name for variable, would `is_ver_13` be better?

Reply:
Good suggestion